CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+To make contributions to this charm, you'll need a working [development setup](https://juju.is/docs/sdk/dev-setup).
+
+You can create an environment for development with `tox`:
+
+```shell
+tox devenv -e integration
+source venv/bin/activate
+```
+
+## Testing
+
+This project uses `tox` for managing test environments. There are some pre-configured environments
+that can be used for linting and formatting code when you're preparing contributions to the charm:
+
+```shell
+tox run -e format        # update your code according to linting rules
+tox run -e lint          # code style
+tox run -e unit          # unit tests
+tox run -e integration   # integration tests
+tox                      # runs 'format', 'lint', and 'unit' environments
+```
+
+## Set up your development environment
+### Install Microk8s
+```
+# Install Microk8s from snap:
+sudo snap install microk8s --classic --channel=1.25
```

Comment:
Change for `strict` since default juju (3.1) requires it.

Reply:
Thanks, updated in separate PR